### ISSUE TITLE
Transcript segments should receive keyboard focus and interaction

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1864]
### Expected behavior

The text segments in the transcript panel should be navigable by keyboard (i.e., receive keyboard focus, and be "clickable" via keyboard Return).

### Current behavior

After selecting the Transcript tab, keyboard focus skips from the Sidebar vertical tab group to the Media player/canvas, skipping over all the Transcript content.

### Steps to reproduce

Use Tab/Shift+Tab and Return keys to navigate through the viewer.

### Possible approach

Add `tabindex="0"` to the `<p>` elements to make them keyboard focusable. NB: this does not also make the items "clickable" via keyboard Return.

**Comment by mjgiarlo** [comment]
Using links (`<a>`) has drawbacks as does using buttons for this approach.
